distributed/comm/registry.py

```diff
@@ -60,10 +60,32 @@ def get_local_address_for(self, loc):
 def get_backend(scheme):
     """
     Get the Backend instance for the given *scheme*.
+
+    New backends can be registered using setuptools entry_points, e.g.:
+    setup(..., entry_points={
+            'distributed.comm.backends': [
+                'udp=dask_udp.backend:UDPBackend',
+            ]
+        },
+    )
```

Comment:
Perhaps show the usage after this. It'd be

```python
>>> get_backend("udp")  # doctest: +SKIP
```

right?

Reply:
I think the calls to get_backend are all internal to distributed at the moment. I think the documentation in communications.rst should be sufficient at this point.  But let me know if you still want the usage in here.